manifests/etcd-defrag/cronjob.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: etcd-defrag
+spec:
+  schedule: "0 0 * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: etcd-defrag
```

Comment:
Is defrag actually necessary? According to https://etcd.io/docs/v3.2/op-guide/maintenance/ :
`Compacting old revisions internally fragments etcd by leaving gaps in backend database. Fragmented space is available for use by etcd but unavailable to the host filesystem.`

Reply:
Yes, correct. Without defragmentation the problem is the free space on the host filesystem.

> After compacting the keyspace, the backend database may exhibit internal fragmentation. Any internal fragmentation is space that is free to use by the backend but still consumes storage space. The process of defragmentation releases this storage space back to the file system.

So defragmentation is needed to free up the disk space on the host filesystem.

Reply:
But the host filesystem in this case is a kubernetes volume used exclusively by etcd. Why do we need to release it?